### ISSUE TITLE
instarepl broken

### ISSUE THREAD

**Issue by anshbansal** [LightTable/LightTable#1979]
I was going through the tutorial for instarepl and I found that it is broken 

Environment - Ubuntu 14.04 64-bit
Light Table version - 0.7.2
Binary version - 0.8.6

![instarepl](https://cloud.githubusercontent.com/assets/4127841/9856140/9279b0f8-5b30-11e5-8083-741ce7c4c6bd.png)

In the image if I change the (+ x y) line then the numbers instead of changing increase. I mean it is like this. See the below image? There are a lot more 3's present then there should be.
![instarepl2](https://cloud.githubusercontent.com/assets/4127841/9856192/df0f2c04-5b30-11e5-9495-d32a6ce6b84b.png)

Also it seems the colouring scheme that was mentioned in the video tutorial is not working either.

**Comment by anshbansal** [comment]
I thought that this was a really cool thing that I can present to some people but seeing the instarepl broken is a complete bummer. 

I am learning clojure so probably going to use this till I can find time for Emacs. I wish this was not broken  as I seriously think that this is really nice.

**Comment by rundis** [comment]
This is fixed in the latest clojure plugin (0.1.3 currently).
- ctrl + space  (open command bar)
- "Plugins: Show plugin manager"
- Find the Clojure plugin and click upgrade

After it's installed, unfortunately you current need to reload it's behaviors or restart LT
Reloading:
- ctrl + space
- "App - Reload behaviors"

Once that is done you can open an instarepl and results shouldn't be stacking any more

**Comment by anshbansal** [comment]
Thanks for the details. Different colors not working. But maybe that is not important.

**Comment by rundis** [comment]
The video's are getting pretty dated, 

Bug reports really need to state what is wrong rather than referring to tutorials for us to be able to consider them. (on a side note, the tutorials page probably needs a disclaimer that things might have changed in later version, alternatively we remove them altogether and refer to them as external non supported learning resources).


Closing this issue. Let me know if 0.1.3 doesn't solve the issue with results-stacking.

**Comment by kleinfelter** [comment]
"Find the Clojure plugin and click upgrade"

This would be really cool, if there were an "upgrade" to click on.  

When I open the plugin manager and I hover over Clojure 0.1.0, I get a little popup menu on the right, with the choices "uninstall" and "source".  I wonder where I might find an "upgrade" to click on.

**Comment by kenny-evitt** [comment]
@kleinfelter Please create a new issue for your problem.
